src/user-managerment/module/individual/pages/IndividualEditProfile.js

```diff
@@ -12,8 +12,10 @@ const IndividualEditProfile = () => {
   return (
     <PageWrapper>
       <div className="w-full h-full flex flex-col gap-8 py-5 overflow-auto">
```

Comment:
If the file is deleted remove it form d prodject

Reply:
That one wasn't deleted.